db/migrate/20181002173218_create_resources.rb

```diff
@@ -0,0 +1,16 @@
+class CreateResources < ActiveRecord::Migration[5.2]
+  def change
+    create_table :resources do |t|
+      t.string :title
+      t.string :type
+      t.string :vendor
+      t.text :url
+      t.text :license_link
+      t.string :user_id
```

Comment:
so you should have a user model and if users can access multiple resources then you will have another model that maintains that relationship. Maybe a `MemberLibrary` or a `UserResources` model. This will have a user_id and a resource_id